app/src/main/res/values-fr/strings.xml

```diff
@@ -249,6 +250,13 @@
     <string name="map_event_title">Carte d\'évènements</string>
     <string name="map_content_description_center_on_user">Centrer sur l\'utilisateur</string>
 
+    /**
+    * package: preferences
+    */
```

Comment:
This is again opinionated, but in XML `/** */` comments are treated as plain text and are not actual comments, so it might be preferable to replace these with `<!-- -->` comments, even though the text you wrote here is ignored by the Android strings parser and thus makes literally no difference.

Reply:
I think this way separates the packages from the screens better, and it’s easier navigating through the file like this, don’t you agree?